app/models/automate_workspace.rb

```diff
@@ -0,0 +1,17 @@
+class AutomateWorkspace < ApplicationRecord
+  include UuidMixin
+  belongs_to :user
+  belongs_to :tenant
+  validates :tenant, :presence => true
+  validates :user, :presence => true
+
+  def merge_output!(hash)
+    if hash['workspace'].blank? && hash['state_vars'].blank?
```

Comment:
Should this be an `||`?  Based on the error message it should fail if either one is blank.  Additionally, if `state_vars => {}` that would technically be blank as well, so maybe this should check for nil instead?